src/main/config.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <sstream>
+#include "gflags/gflags.h"
+#include "config.hpp"
+
+extern "C" {
+	#include "exec_monitor.h"
+}
+
+/*
+	FLAGS SECTION
+*/
+
+// EXEC FLAG - MONITOR PROCESS EXECUTION
+DEFINE_bool(exec, false, "Monitors process executions.");
```

Comment:
If you were to change this flag with something along the line of --monitor=exec, you could make that flag the only mandatory flag which would be easier to explain in the usage help, also it would be easier to determine what usecase is picked and you could apply validation similar to what you do with save and format. What do you think?